### PR TITLE
Options in the duplicate rule modal updated

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/3220. 

Preview here.

### CODE REVIEW

**Comment by nastasha-solomon** [comment]
@elasticmachine run elasticsearch-ci/docs

**Review by benironside — APPROVED** [review]
LGTM!

File: docs/detections/rules-ui-manage.asciidoc [hunk 1]
```diff
@@ -151,7 +151,7 @@ NOTE: The action frequency you select applies to all actions (both new and exist
 
 You can duplicate, enable, disable, and delete rules:
 
-NOTE: When duplicating a rule that has exceptions, you can choose to duplicate the rule and its exceptions or duplicate the rule only. If you choose to duplicate the rule and its exceptions, copies of the exceptions are created and added to the duplicated rule's <<detections-ui-exceptions,default rule list>>. In addition, if the original rule uses exceptions from a shared exception list, the duplicated rule will reference the same shared exception list.  
+NOTE: When duplicating a rule that has exceptions, you can choose to duplicate the rule and its exceptions (active and expired), the rule and active exceptions only, or only the rule. If you choose to duplicate the rule and its exceptions, copies of the exceptions are created and added to the duplicated rule's <<detections-ui-exceptions,default rule list>>. If the original rule used exceptions from a shared exception list, the duplicated rule will reference the same shared exception list.
```
Comment (jmikell821) [inline docs/detections/rules-ui-manage.asciidoc:154]:
```suggestion
NOTE: When duplicating a rule with exceptions, you can choose to duplicate the rule and its exceptions (active and expired), the rule and active exceptions only, or only the rule. If you duplicate the rule and its exceptions, copies of the exceptions are created and added to the duplicated rule's <<detections-ui-exceptions,default rule list>>. If the original rule used exceptions from a shared exception list, the duplicated rule will reference the same shared exception list.  
```

**Review by jmikell821 — COMMENTED** [review]
Just a slight suggestion. Looks good, thanks!

**Pull request merged**